common/pb/serverinfo_user.proto

```diff
@@ -11,6 +12,10 @@ message ServerInfo_User {
         Male = 0;
         Female = 1;
     };
+    enum UserRole {
```

Comment:
I'm not a fan of this - I think it makes the roles hard to extend. I'll post my concept on the main pr section